lib/test/integration/integration_test_helpers.go

```diff
@@ -331,7 +332,21 @@ func initInput(t *testing.T, env *testEnvironment) types.Input {
 	require.NoError(t, err)
 	assert.Empty(t, lints)
 
-	input, err := input.New(s.Input, types.NoopMgr(), env.log, env.stats)
+	caches := map[string]types.Cache{}
+	for name, c := range s.Manager.Caches {
+		cacheObj, err := cache.New(c, types.NoopMgr(), env.log, env.stats)
+		require.NoError(t, err)
+		caches[name] = cacheObj
+	}
+
+	var mgr types.Manager
+	if len(caches) > 0 {
```

Comment:
Rather than using a mock manager here it's probably worth instantiating a real one, you can just use noops for all the params (and nil for api) like this: `manager.New(s.Manager, nil, log.Noop(), metrics.Noop())`.